content/install.md

```diff
@@ -1,9 +1,18 @@
 # Install the Wazo Platform UC use case
 
+## Requirements
+
+For a small install of about 20 users, less than 20 calls per day:
+
+- RAM: 2 GiB with 1 GiB swap
+- Storage: 2.5 GiB of storage is a very tight minimum, 8 GiB is comfortable
+
+## Procedure
+
 To install the Unified Communication use case in an all-in-one
 setup, do the following steps:
 
-1. Install a Debian 10 Buster system
+1. Install a Debian 10 Buster system with a default locale with an UTF-8 charset.
```

Comment:
It looks it's critical, maybe we should add a ansible check on that.

Reply:
there is already one ;-)